servatrice/src/servatrice.cpp

```diff
@@ -405,6 +405,8 @@ bool Servatrice::initServer()
         }
     }
 #endif
+
+    qDebug() << "Idle client timeout value: " << getIdleClientTimeout();
```

Comment:
This provides no information about which user this relates to. That will make this log hard to use in any way for the admin